### ISSUE TITLE
Version of VirtualBox installed that is not supported.

### ISSUE THREAD

**Issue by ceah** [mitchellh/vagrant#5381]
Hi, 

Trying to fix not being able to connect to my local site via vagrant. I upgraded VirtualBox and now get message:

Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1

Not sure what steps take. I have the latest update 4.3.22 VB

This is my debug message below, thanks

vagrant@precise32:~$ VAGRANT_LOG=debug vagrant up
 INFO global: Vagrant version: 1.0.1
 INFO vagrant: `vagrant` invoked: ["up"]
DEBUG vagrant: Creating Vagrant environment
 INFO environment: Environment initialized (#<Vagrant::Environment:0xb7303fb0>)
 INFO environment:   - cwd: /home/vagrant
 INFO environment: Home path: /home/vagrant/.vagrant.d
 INFO plugin: Searching and loading any available plugins...
 INFO plugin: Loaded 0 plugins.
DEBUG vagrant: Loading environment
 INFO environment: Loading configuration...
DEBUG loader: Set :default = "/usr/share/vagrant/config/default.rb"
DEBUG loader: Populating proc cache for "/usr/share/vagrant/config/default.rb"
DEBUG loader: Load procs for pathname: #<Pathname:/usr/share/vagrant/config/default.rb>
DEBUG loader: Set :root = #<Pathname:/home/vagrant/Vagrantfile>
DEBUG loader: Populating proc cache for #<Pathname:/home/vagrant/Vagrantfile>
DEBUG loader: Load procs for pathname: #<Pathname:/home/vagrant/Vagrantfile>
DEBUG loader: Loading configuration in order: [:default, :box, :home, :root, :vm]
DEBUG loader: Loading from: default (evaluating)
DEBUG loader: Loading from: root (evaluating)
DEBUG loader: Configuration loaded successfully
DEBUG environment: Loading configuration for VM: default
DEBUG loader: Set :default = "/usr/share/vagrant/config/default.rb"
DEBUG loader: Set :root = #<Pathname:/home/vagrant/Vagrantfile>
DEBUG loader: Set :vm = []
DEBUG loader: Loading configuration in order: [:default, :box, :home, :root, :vm]
DEBUG loader: Loading from: default (cache)
DEBUG loader: Loading from: root (cache)
DEBUG loader: Configuration loaded successfully
DEBUG loader: Set :default = "/usr/share/vagrant/config/default.rb"
DEBUG loader: Set :root = #<Pathname:/home/vagrant/Vagrantfile>
DEBUG loader: Set :vm = []
DEBUG loader: Loading configuration in order: [:default, :box, :home, :root, :vm]
DEBUG loader: Loading from: default (cache)
DEBUG loader: Loading from: root (cache)
DEBUG loader: Configuration loaded successfully
 INFO cli: CLI: [] "up" []
DEBUG cli: Invoking command class: Vagrant::Command::Up []
DEBUG up: 'Up' each target VM...
 INFO datastore: Created: /home/vagrant/.vagrant
 INFO virtualbox_base: VBoxManage path: VBoxManage
 INFO subprocess: Starting process: ["VBoxManage", "--version"]
DEBUG subprocess: Selecting on IO
DEBUG subprocess: stdout: WARNING: The character device /dev/vboxdrv does not exist.
	 Please install the virtualbox-ose-dkms package and the appropriate
	 headers, most likely linux-headers-generic.

	 You will not be able to start VMs until this problem is fixed.

DEBUG subprocess: stdout: 4.1.12_Ubuntur77245

DEBUG subprocess: Waiting for process to exit. Remaining to timeout: 32000
DEBUG subprocess: Exit status: 0
DEBUG virtualbox: Finding driver for VirtualBox version: WARNING: The cha
ERROR vagrant: Vagrant experienced an error! Details:
ERROR vagrant: #<Vagrant::Errors::VirtualBoxInvalidVersion: Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1>
ERROR vagrant: Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1
ERROR vagrant: /usr/lib/ruby/vendor_ruby/vagrant/driver/virtualbox.rb:60:in `initialize'
/usr/lib/ruby/vendor_ruby/vagrant/vm.rb:129:in `new'
/usr/lib/ruby/vendor_ruby/vagrant/vm.rb:129:in `reload!'
/usr/lib/ruby/vendor_ruby/vagrant/vm.rb:35:in `initialize'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:434:in `new'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:434:in `load_vms!'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:433:in `each'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:433:in `load_vms!'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:125:in `vms'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:158:in `multivm?'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:133:in `vms_ordered'
/usr/lib/ruby/vendor_ruby/vagrant/command/base.rb:91:in `with_target_vms'
/usr/lib/ruby/vendor_ruby/vagrant/command/up.rb:24:in `execute'
/usr/lib/ruby/vendor_ruby/vagrant/cli.rb:38:in `execute'
/usr/lib/ruby/vendor_ruby/vagrant/environment.rb:167:in `cli'
/usr/bin/vagrant:43
 INFO interface: error: Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1
Vagrant has detected that you have a version of VirtualBox installed
that is not supported. Please install one of the supported versions
listed below to use Vagrant:

4.0, 4.1
vagrant@precise32:~$

**Comment by matt-richardson** [comment]
Hi @ceah. Looks like you've got a really old version of vagrant (1.0.1). How did you install it?
Latest available version is 1.7.2

**Comment by ceah** [comment]
Ha! That's weird, earlier this morning I did the vagrant -v and had 1.7.2. 
I tried to start today with the usual vagrant up and got this:

==> default: Checking if box 'hashicorp/precise32' is up to date...
==> default: VirtualBox VM is already running.
==> default: Checking for host entries
 and couldn't access my local site. 

After going around in circles I got a message that said this:

 default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.

And it all went downhill from there. 

Maybe I will try to install 1.7.2 again and go from there.

Thanks for the quick response, btw, appreciate it!
